### ISSUE TITLE
Parity installation fails

### ISSUE THREAD

**Issue by 21xhipster** [paritytech/homebrew-paritytech#27]
```
==> multirust run stable cargo build --release
Last 15 lines from /Users/xhipster/Library/Logs/Homebrew/parity/02.multirust:
   Compiling hpack v0.2.0
   Compiling unicode-bidi v0.2.3
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-m64" "-L" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib" "/private/tmp/parity-20161009-13836-1ycx8cv/target/release/build/heapsize-b8189d5c83f95113/build_script_build.0.o" "-o" "/private/tmp/parity-20161009-13836-1ycx8cv/target/release/build/heapsize-b8189d5c83f95113/build_script_build" "-Wl,-dead_strip" "-nodefaultlibs" "-L" "/private/tmp/parity-20161009-13836-1ycx8cv/target/release/deps" "-L" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/libstd-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/libunwind-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/librand-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/libcollections-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/librustc_unicode-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/liballoc-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/liballoc_jemalloc-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/liblibc-40393716.rlib" "/private/tmp/parity-20161009-13836-1ycx8cv/.brew_home/.multirust/toolchains/stable/lib/rustlib/x86_64-apple-darwin/lib/libcore-40393716.rlib" "-l" "System" "-l" "pthread" "-l" "c" "-l" "m" "-l" "compiler-rt"
  = note: ld: unexpected token: !tapi-tbd-v2 file '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/usr/lib/libSystem.tbd' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)


error: aborting due to previous error

Build failed, waiting for other jobs to finish...
error: Could not compile `heapsize`.
```

Can somebody help?

**Comment by rachimo34** [comment]
same here
==> Cloning https://github.com/ethcore/parity.git
Updating /Users/orisproduction/Library/Caches/Homebrew/parity--git
==> Checking out tag v1.3.8
==> multirust update stable
Last 15 lines from /Users/orisproduction/Library/Logs/Homebrew/parity/01.multirust:

shasum: WARNING: 1 computed checksum did NOT match
rustup: command failed: shasum -c -a 256 /private/tmp/parity-20161017-11440-1l24j18/.brew_home/.multirust/rustup/tmp/tmp-12128-9/tmpsums
rustup: checksum failed for '/private/tmp/parity-20161017-11440-1l24j18/.brew_home/.multirust/rustup/dl/e23857d57cbcd86ffdc7/rust-1.12.0-x86_64-apple-darwin.tar.gz'

**Comment by jwter** [comment]
same here...how solve??
jwter-WIFI:proto jwter$ brew install parity
==> Installing parity from ethcore/ethcore
==> Cloning https://github.com/ethcore/parity.git
Updating /Users/jwter/Library/Caches/Homebrew/parity--git
==> Checking out branch stable
==> multirust update stable
Last 15 lines from /Users/jwter/Library/Logs/Homebrew/parity/01.multirust:
2016-11-02 14:34:42 +0800

multirust update stable

multirust: installing toolchain 'stable'
rustup: gpg not available. signatures will not be verified
rustup: downloading manifest for 'stable'
rustup: command failed: curl -s -f -O https://static.rust-lang.org/dist/channel-rust-stable.toml.sha256
rustup: couldn't download checksum file 'https://static.rust-lang.org/dist/channel-rust-stable.toml.sha256'
rustup: downloading manifest for 'stable'
rustup: command failed: curl -s -f -O https://static.rust-lang.org/dist/channel-rust-stable.sha256
rustup: couldn't download checksum file 'https://static.rust-lang.org/dist/channel-rust-stable.sha256'

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/ethcore/homebrew-ethcore/issues

These open issues may also help:
Parity installation fails https://github.com/ethcore/homebrew-ethcore/issues/27

**Comment by chevdor** [comment]
Having this issue as well. I started with `brew uninstall --force parity`followed by `brew update && brew install parity --beta`

```
Updated 1 tap (caskroom/cask).
==> Installing parity from ethcore/ethcore
==> Cloning https://github.com/ethcore/parity.git
Updating /Users/will/Library/Caches/Homebrew/parity--git
==> Checking out tag v1.4.2
==> multirust update stable
Last 15 lines from /Users/will/Library/Logs/Homebrew/parity/01.multirust:
2016-11-11 11:58:04 +0100

multirust update stable


If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/ethcore/homebrew-ethcore/issues

These open issues may also help:
Parity installation fails https://github.com/ethcore/homebrew-ethcore/issues/27
```

**Comment by aribo** [comment]
I'm having this issue on a Mac trying to install 1.3.4

> multirust: installing toolchain 'stable'
> rustup: gpg not available. signatures will not be verified
> rustup: downloading manifest for 'stable'
> rustup: downloading toolchain for 'stable'
> ####################################                                      50.6%
> curl: (56) SSLRead() return error -9806
> rustup: command failed: curl -# -C - -f -O https://static.rust-lang.org/dist/2016-11-08/rust-1.13.0-x86_64-apple-darwin.tar.gz
> rustup: couldn't download 'https://static.rust-lang.org/dist/2016-11-08/rust-1.13.0-x86_64-apple-darwin.tar.gz'
> 
> If reporting this issue please do so to (not Homebrew/brew or Homebrew/core):
> ethcore/ethcore
> 
> These open issues may also help:
> Parity installation fails https://github.com/ethcore/homebrew-ethcore/issues/27

**Comment by caoxg** [comment]
Updating Homebrew...
==> Installing parity from ethcore/ethcore
==> Cloning https://github.com/ethcore/parity.git
Updating /Users/xiaogang/Library/Caches/Homebrew/parity--git
==> Checking out tag v1.3.14
==> multirust update stable
==> multirust run stable cargo build --release
Last 15 lines from /Users/xiaogang/Library/Logs/Homebrew/parity/02.multirust:
 Downloading time v0.1.35
 Downloading isatty v0.1.1
 Downloading number_prefix v0.2.6
 Downloading lazy_static v0.2.1
 Downloading env_logger v0.3.3
 Downloading ansi_term v0.7.2
 Downloading semver v0.2.3
 Downloading typeable v0.1.2
 Downloading unicase v1.4.0
 Downloading language-tags v0.2.2
 Downloading url v0.5.9
error: unable to get packages from source

Caused by:
  failed to get 200 response from `https://crates.io/api/v1/crates/url/0.5.9/download`, got 302

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/ethcore/homebrew-ethcore/issues

These open issues may also help:
Parity installation fails https://github.com/ethcore/homebrew-ethcore/issues/27

**Comment by chtmorris** [comment]
I'm getting this issue trying to brew install Parity:

error: Package `parity v1.4.10 (file:///private/tmp/parity-20170120-44929-1jgt9kd)` does not have these features: `final`

**Comment by Aquilae32** [comment]
I have the same error.

On MacOs 10.12.2 (Sierra), I did "brew install parity --beta", which installed version 1.5.0. Then I did "brew unlink parity" in view of installing the stable version 1.4.10, for which I used the command "brew install parity", which then indicated:

==> Installing parity from ethcore/ethcore
==> Cloning https://github.com/ethcore/parity.git
Updating /Users/admin/Library/Caches/Homebrew/parity--git
==> Checking out tag v1.4.10
==> multirust update stable
==> multirust run stable cargo build --release --features final
Last 15 lines from /Users/admin/Library/Logs/Homebrew/parity/02.multirust:
    Updating git repository `https://github.com/ethcore/jsonrpc-http-server.git`
    Updating git repository `https://github.com/ethcore/hyper`
    Updating git repository `https://github.com/ethcore/mio`
    Updating git repository `https://github.com/ethcore/json-tcp-server`
    Updating git repository `https://github.com/ethcore/nanomsg.rs.git`
    Updating git repository `https://github.com/ethcore/js-precompiled.git`
    Updating git repository `https://github.com/ethcore/rust-secp256k1`
    Updating git repository `https://github.com/ethcore/rust-rocksdb`
    Updating git repository `https://github.com/ethcore/rotor`
    Updating git repository `https://github.com/ctz/rustls`
    Updating git repository `https://github.com/ethcore/ws-rs.git`
    Updating git repository `https://github.com/carllerche/bytes`
    Updating git repository `https://github.com/carllerche/slab`
    Updating git repository `https://github.com/carllerche/stable-heap`
error: Package `parity v1.4.10 (file:///private/tmp/parity-20170120-42027-1mat93g)` does not have these features: `final`

**Comment by debris** [comment]
@chtmorris @Aquilae32 `final` issue has been fixed in #35

**Comment by debris** [comment]
we use prebuilt binaries now, closing
